matrices/transformation_matrix.cxx

```diff
@@ -1,5 +1,6 @@
 #include"transformation_matrix.hxx"
-#include<t8_vec.h>
+#include <t8_vec.h>
+#include <t8_mat.h>
```

Comment:
I think that this header isn't needed anymore, right?

Reply:
done